examples/FIDtoEcho_Actual_Challenging.py

```diff
@@ -18,41 +18,71 @@
 from collections import OrderedDict
 from numpy.random import normal
 from scipy.signal import tukey
-init_logging(level='debug')
+init_logging(level="debug")
```

Comment:
you need to run `git merge --no-ff jf_review_actual` and resolve any conflicts.  I have run black on jf_review_actual, so that will clean up this diff, but there are other unmerged differences.

Reply:
(after pulling jf_review_actual to update it)